layouts/partials/header/header.html

```diff
@@ -8,14 +8,11 @@
 
 <header class="navbar navbar-expand-lg navbar-light doks-navbar">
   <nav class="container-xxl justify-content-start" aria-label="Main navigation">
-    <button class="btn btn-menu d-block d-lg-none order-0 m-0" type="button" data-bs-toggle="offcanvas" data-bs-target="#offcanvasDoks" aria-controls="offcanvasDoks" aria-label="Open main menu">
-      <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-menu"><line x1="3" y1="12" x2="21" y2="12"></line><line x1="3" y1="6" x2="21" y2="6"></line><line x1="3" y1="18" x2="21" y2="18"></line></svg>
-    </button>
-    <a class="navbar-brand p-0 me-auto coh-title" href="{{ .Site.Home.Permalink }}" aria-label="{{ .Site.Params.Title }}">{{ .Site.Params.Title }}</a>
+    {{ partial "header/logo.html" . }}
 
-    <div class="offcanvas offcanvas-start border-0 py-md-1 order-1" tabindex="-1" id="offcanvasDoks" data-bs-backdrop="true" aria-labelledby="offcanvasDoksLabel">
-      <div class="offcanvas-header d-lg-none">
-        <h2 class="h5 offcanvas-title ps-2" id="offcanvasDoksLabel"><a class="text-dark" href="{{ .Site.Home.Permalink }}">{{ .Site.Params.Title }}</a></h2>
+    <div class="offcanvas offcanvas-start border-0 py-1 order-1" tabindex="-1" id="offcanvasDoks" data-bs-backdrop="true" aria-labelledby="offcanvasDoksLabel">
+      <div class="offcanvas-header d-lg-none" style="padding-left:1.5rem; padding-top:0.25rem; padding-bottom: 0.5rem;">
```

Comment:
Just pointing out that these (and the ones in layouts/search/list.html) styles are used with `style=` instead of a dedicated class. Since they won't be changed ever, you can leave them like that.